app/views/activities/index.js.erb

```diff
@@ -0,0 +1,29 @@
+var $activitiesGroupsContainer = $('[data-behavior="activities-groups-container"]');
+
+<% if @activities_groups.present? %>
+  var template = '<%= j render 'activities/activities_groups', activities_groups: @activities_groups %>';
+  var $template = $(template);
+
+  $.each($template, function(index, activitiesGroup) {
+    var $activitiesGroup = $(activitiesGroup);
+    var timeElementDatetime = $activitiesGroup.find('[data-behavior~=activity-day-value]').attr('datetime');
+    var $timeElementInDOM = $activitiesGroupsContainer.find(`[datetime="${timeElementDatetime}"]`);
+    // console.log($timeElementInDOM);
+    if ($timeElementInDOM.length) {
+      var $activitiesGroupContainer = $timeElementInDOM.parents('[data-behavior="activities-group"]');
```

Comment:
Can we use better variable naming here? It's easy to mistake `$activitiesGroupsContainer` and `$activitiesGroupContainer` and they are two very different elements. `$activitiesGroupsContainer` is the parent container while `$activitiesGroupContainer` seems to be an activities group within a specific date.

Reply:
I've updated it to infinite-scroll-container instead, Thanks!